domains/certificates/Shared/Contracts/Certificates/CertificateIssuedInProjectOriginEvent.cs

```diff
@@ -0,0 +1,5 @@
+using System;
+
+namespace Contracts.Certificates;
+
+public record CertificateIssuedInProjectOriginEvent(Guid CertificateId);
```

Comment:
Should we use the term "ProjectOrigin" here or should it be "Registry" instead?

Reply:
corrected